src/page.js

```diff
@@ -0,0 +1,135 @@
+// eslint-disable-next-line
+export const htmlPage = () => {
+  const scores = [
+
+    {
+      id: 1,
+      name: 'name',
+      score: 100,
+    },
+    {
+      id: 2,
+      name: 'name',
+      score: 100,
+    },
+    {
+      id: 3,
+      name: 'name',
+      score: 100,
+    },
+    {
+      id: 4,
+      name: 'name',
+      score: 100,
+    },
+    {
+      id: 5,
+      name: 'name',
+      score: 100,
+    },
+    {
+      id: 6,
+      name: 'name',
+      score: 100,
+    },
+    {
+      id: 7,
+      name: 'name',
+      score: 100,
+    },
+  ];
+
+  // set page title
+  document.title = 'Leaderboard';
+
+  // page header
+  const header = document.createElement('header');
+  const headerTitle = document.createElement('h1');
+  const titleText = document.createTextNode('Leaderboard');
+  headerTitle.appendChild(titleText);
+  header.appendChild(headerTitle);
+  document.body.appendChild(header);
+
+  // section
+  const section = document.createElement('section');
+  section.className = 'container';
+
+  // div for table
+  const divTable = document.createElement('div');
+  divTable.className = 'divTable';
+
+  const divTableHeader = document.createElement('div');
+  divTableHeader.className = 'tableHeader';
+
+  const tableTitle = document.createElement('h2');
+  tableTitle.className = 'tableTitle';
+
+  const tableTitleText = document.createTextNode('Recent score');
+  tableTitle.appendChild(tableTitleText);
+  const tableRefreshButton = document.createElement('button');
+  tableRefreshButton.className = 'refreshBtn';
+
+  tableRefreshButton.innerHTML = 'Refresh';
```

Comment:
- [ ] Again, based on the requirements of this project, when the refresh button is clicked, it should hit the GET API endpoint, causing the scores list to change. You haven't yet implemented this. Let's once more check the general requirements of this project and work on this.

Reply:
Thanks! I made a mistake, this pull request was for a basic setup project, I submitted it by mistake. but I will follow the recommendations and complete what is missing. Thank you!